core/domain/activity_jobs_one_off.py

```diff
@@ -215,15 +215,23 @@ def entity_classes_to_map_over(cls):
     def map(commit_model):
         class_name = commit_model.__class__.__name__
         last_updated = commit_model.last_updated
+        created_on = commit_model.created_on
         if (FixCommitLastUpdatedOneOffJob.MIGRATION_START < last_updated <
                 FixCommitLastUpdatedOneOffJob.MIGRATION_END):
             commit_model.last_updated = commit_model.created_on
             commit_model.put(update_last_updated_time=False)
             yield ('SUCCESS_FIXED - %s' % class_name, commit_model.id)
+        elif (datetime.timedelta(0) < last_updated - created_on <
+              datetime.timedelta(7)):
+            yield ('SUCCESS_CORRECT - %s' % class_name, commit_model.id)
```

Comment:
Let's change this to SUCCESS_NEWLY_CREATED. Maybe on reflection we can make the time diff smaller -- perhaps 1 hour or less. (That's what we'd expect for a newly-minted model, right?)

Also, given what you said here:

> Yep, although I think that many commit log models that have system admin as a committer are also correct (these weren't migrated and there is quite a few of them), so I have only included more info for the 'incorrect' commit log models.

let's maybe also add a check for "if admin is committer" and add a SUCCESS_ADMIN case. That way we only surface models where we don't expect the given results.

Reply:
Done.